csm/cli/support_bundle/csm_bundle_generate.py

```diff
@@ -41,13 +44,19 @@ async def init(command):
         csm_log_directory_path = Conf.get(const.CSM_GLOBAL_INDEX, "Log.log_path")
         uds_log_directory_path = Conf.get(const.CSM_GLOBAL_INDEX, "Log.uds_log_path")
         elasticsearch_log_path = Conf.get(const.CSM_GLOBAL_INDEX, "Log.elasticsearch_log_path")
+        alerts_file_path = Conf.get(const.CSM_GLOBAL_INDEX, "SUPPORT_BUNDLE.alerts_file_path")
+        # Fetch alerts for support bundle.
+        alerts_data = await CSMBundle.fetch_and_save_alerts()
```

Comment:
This is Fetching Alerts every time even for `csm`, `uds` and `es` bundles are generated.
Please add a check that is components is Alerts then only the following lines are to be executed.

Reply:
implemented review comments